SystemInfo/Value.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * File containing the Value class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\SystemInfo;
+
+use eZ\Publish\API\Repository\Values\ValueObject;
+
+/**
+ * System information Value.
+ */
+abstract class Value extends ValueObject
```

Comment:
An interface would work as well, as a matter of fact.

But I would maybe not name it `Value` alone. I'd even name the interface `Value\SystemInfo`.